tests/MongoDB.Analyzer.Tests/Infrastructure/PathUtilities.cs

```diff
@@ -20,17 +20,17 @@ internal static class PathUtilities
 {
     private static readonly string s_testCasesBaseNamespace = "MongoDB.Analyzer.Tests.Common.TestCases";
     private static readonly string s_testCasesBaseFolder = s_testCasesBaseNamespace;
-    private static readonly string s_projectParentFolderPrefix = @"..\..\..\..\";
+    private static readonly string s_projectParentFolderPrefix = @"../../../../";
```

Comment:
I will validate that this changes work on windows. If not, we'll need to resolve the path based on OS.